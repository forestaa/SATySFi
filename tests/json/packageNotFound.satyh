@require: notfound

let x = 1
