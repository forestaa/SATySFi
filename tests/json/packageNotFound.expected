{
  "file": "",
  "range": {
    "start": { "line": 0, "character": 0 },
    "end": { "line": 0, "character": 0 }
  },
  "message":
    "! [Error] package file not found:\n      notfound\n    candidate paths:\n      /mnt/c/Users/tdaic/build/SATySFi/.satysfi/local/packages/notfound.satyh\n      /mnt/c/Users/tdaic/build/SATySFi/.satysfi/local/packages/notfound.satyg\n      /home/foresta/.satysfi/local/packages/notfound.satyh\n      /home/foresta/.satysfi/local/packages/notfound.satyg\n      /usr/local/share/satysfi/local/packages/notfound.satyh\n
 /usr/local/share/satysfi/local/packages/notfound.satyg\n      /usr/share/satysfi/local/packages/notfound.satyh\n      /usr/share/satysfi/local/packages/notfound.satyg\n      /mnt/c/Users/tdaic/build/SATySFi/.satysfi/dist/packages/notfound.satyh\n      /mnt/c/Users/tdaic/build/SATySFi/.satysfi/dist/packages/notfound.satyg\n
  /home/foresta/.satysfi/dist/packages/notfound.satyh\n      /home/foresta/.satysfi/dist/packages/notfound.satyg\n      /usr/local/share/satysfi/dist/packages/notfound.satyh\n      /usr/local/share/satysfi/dist/packages/notfound.satyg\n      /usr/share/satysfi/dist/packages/notfound.satyh\n      /usr/share/satysfi/dist/packages/notfound.satyg"
}
